sdk/02_tutorial/07_verification.md

```diff
@@ -11,59 +11,71 @@ import TabItem from '@theme/TabItem';
 import Verify from '!!raw-loader!@site/code_examples/sdk_examples/src/workshop/verify.ts';
 import CreatePresentation from '!!raw-loader!@site/code_examples/sdk_examples/src/workshop/holder/createPresentation.ts';
 
+<!-- Verficiation criteria, expect it  to be tied to Verifier DID, tied to the verficationCriteria, based on four options -->
+
 In this section, you play the role of a <span className="label-role verifier">Verifier</span> that does the following:
 
-1. Take a `Presentation` object supplied by a <span className="label-role claimer">Claimer</span>
+:::info
+
+The Verifier also needs an account and DID, but the tutorial omits these steps.
+
+:::
+
+1. Take a `VerifiablePresentation` object supplied by a <span className="label-role holder">Holder</span>
 2. Verify that its data is correct
-3. Verify that the attestation is valid, i.e., its hash exists on-chain and the attestation has not been revoked
-4. Verify that the <span className="label-role claimer">Claimer</span> sending the `Credential` owns it
+3. Verify that the presentation is valid, i.e., its hash exists on-chain and the presentation hasn't been revoked
+4. Verify that the <span className="label-role holder">Holder</span> sending the `Credential` owns it
 
-:::info Presentation object
+:::info VerifiablePresentation object
 
-The <span className="label-role claimer">Claimer</span> uses a Credential to create the `Presentation` object.
-Unlike the credential, a `Presentation` can hide some attributes that are not required by the <span className="label-role verifier">Verifier</span> and can contain a claimer-signed challenge.
-A `Presentation` also contains a proof that the <span className="label-role claimer">Claimer</span> owns the credential.
+The <span className="label-role holder">Holder</span> uses a Credential to create the `VerifiablePresentation` object.
+Unlike the credential, a `VerifiablePresentation` can hide some attributes that aren't required by the <span className="label-role verifier">Verifier</span> and can contain a holder-signed challenge.
```

Comment:
a Verifiable Presentation actually has an array of credentials, and it's in these credentials that some properties are hidden. so this is not entirely accurate and might be confusing

Reply:
@rflechtner So is the only change here object > array? Or something else?

Reply:
The main functional change is that one, yes; in the old Kilt a presentation was simply a credential that was signed by the holder, plus presentation-related data such as a challenge. The signature and challenge and all other presentation-related data is now held by the Verifiable Presentation object, which can have one or more credentials nested within. So while you had one presentation per credential prior, you can now present multiple at the same time using only one signature.

The data model of the presentation has changed significantly though; we now use the data model described in https://www.w3.org/TR/vc-data-model/#presentations-0